pages/en/lb3/3.0-Release-Notes.md

```diff
@@ -30,6 +30,8 @@ dependency.
 
 See [loopback#275](https://github.com/strongloop/loopback/issues/275).
 
+Please see [how to migrate](http://loopback.io/doc/en/lb3/Migrating-to-3.0.html#loopback-datasource-juggler-is-a-regular-dependency-now)
```

Comment:
As long as we're nitpicking, you forgot to downcase the "See" when you add "Please" in a few places.